batchflow/research/named_expr.py

```diff
@@ -91,15 +91,32 @@ def get(self, **kwargs):
         return job.ids[unit.index]
 
 class REP(ResearchNamedExpression): # ResearchExperimentPath
-    """ NamedExpression for path to the current experiment """
+    """ NamedExpression for path to folder corresponding to the current config """
+    def __init__(self, name=None, relative=False):
+        """ NamedExpression for path inside to experiment folder.
+
+        Parameters
+        ----------
+        name : str or None
+            NamedExpression name
+        relative : bool
+            if True, absolute path including name of the root research folder,
+            if False, the path inside of the research folder.
+        """
+        super().__init__(name)
+        self.relative = relative
+
     def _get(self, **kwargs):
         _, kwargs = super()._get(**kwargs)
-        return kwargs['job'], kwargs['experiment']
+        return kwargs['job'], kwargs['experiment'], kwargs['path']
 
     def get(self, **kwargs):
-        job, experiment = self._get(**kwargs)
+        job, experiment, path = self._get(**kwargs)
         unit = list(experiment.values())[0]
```

Comment:
```suggestion
        unit = next(iter(experiment.values()))  # all units in experiment have same experiment_path
```
this avoids temporary list creation